tests/unit/test_datasets.py

```diff
@@ -74,12 +74,14 @@ def test_fetch(self, datasets):
         datasets.return_value.downloader.download.assert_called_once_with('file.xz')
 
     @patch('serenata_toolbox.datasets.Datasets')
-    def test_fetch(self, datasets):
-        fetch('file.xz', 'test')
-        datasets.assert_called_once_with('test')
-        datasets.return_value.downloader.download.assert_called_once_with('file.xz')
-
-    @patch('serenata_toolbox.datasets.Datasets')
     def test_fetch_latest_backup(self, datasets):
         fetch_latest_backup('test')
         self.assertTrue(datasets.return_value.downloader.download.called)
+
+    @patch('os.path.exists')
+    @patch('serenata_toolbox.datasets.Datasets')
+    def test_fetch_latest_backup_only_when_missing(self, datasets, os_path_exists):
```

Comment:
This test should also be added to the Journey tests don't you think?

cc @cuducos @lipemorais

Reply:
No, I think the proper mocks @tuliocasagrande added make it a typical unit test.